### ISSUE TITLE
telemetry reporter crash error

### ISSUE THREAD

**Issue by computeronix** [Storj/storjshare-cli#176]
### Package Versions

```
StorjShare: 7.0.0-beta
Core:       4.0.0-rc.2
Protocol:   0.8.1
```

```
v4.5.0
```

### Expected Behavior

Telemetry should work

### Actual Behavior

Telemetry does not work (no speedtest file in tmp nor any telemetry logs).  The below error occurs upon node startup.

```
{"level":"error","message":"telemetry reporter failed, reason: Cannot read property 'getNodeID' of undefined","timestamp":"2016-09-21T00:59:02.988Z"}

```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. Install storjshare-cli pre-release
2. Start it
3. Log appears right away (Windows and Linux)

**Comment by bookchin** [comment]
Ah, the telemetry reporter is written for storj-lib@3.x.x so we'll need to update it when 4.0.0 is stable.

**Comment by littleskunk** [comment]
```
/usr/local/lib
└─┬ storjshare-cli@7.0.0
  └── storj-telemetry-reporter@2.0.0
```

```
{"level":"error","message":"telemetry reporter failed, reason: Cannot read property 'getNodeID' of undefined","timestamp":"2016-10-07T13:06:03.417Z"}
```

**Comment by lazerdye** [comment]
I'm still seeing this error with a fresh install of storjshare-cli@7.0.1:

/home/storj/git/storj-next
└─┬ storjshare-cli@7.0.1
....
  └─┬ storj-telemetry-reporter@2.0.0
....

**Comment by aleitner** [comment]
Solved by #198
